packages/common/src/audius-query/createApi.ts

```diff
@@ -143,7 +142,7 @@ const useQueryState = <Args, Data>(
   reducerPath: string,
   endpointName: string,
   endpoint: EndpointConfig<Args, Data>
-) => {
+): Nullable<PerKeyState<any> & { isInitialValue?: boolean }> => {
```

Comment:
does `PerKeyState<Data>` work? Maybe not cause it's like `PerKeyState<Normalized<Data>` really

Reply:
Ah i see i can try that, the main thing was complaining about isInitialValue not being a port of PerStateKey